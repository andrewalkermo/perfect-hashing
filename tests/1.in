i
4
56
joao
20
18
maria
30
27
carlos
15
29
ana
18
p
e