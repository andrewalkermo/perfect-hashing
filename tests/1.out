estrutura de hashing perfeito criada